Include/AndroidExtensions/Globals.h

```diff
@@ -8,7 +8,9 @@
 namespace android::global
 {
     void Initialize(JavaVM* javaVM, jobject appContext);
-    // Passing nullptr for appContext/assetManager clears cached globals during teardown.
+    // Passing nullptr for appContext/assetManager clears cached globals so embedders can
+    // explicitly tear down between reloads, or supply only an asset manager when no stable
+    // Java Context is available (e.g., headless/unit-test harnesses).
     void Initialize(JavaVM* javaVM, jobject appContext, jobject assetManager);
```

Comment:
With the new usage, we would call Initialize with nullptr to tear down. This seems awkward. Maybe it be better to create a separate function called Uninitialize?

Reply:
You're right - explicit teardown isn't actually needed. The real use cases are:
 * supporting headless scenarios  without a Context
*  making re-initialization safe for hot reloads. 
 
 The cleanup happens automatically at the start of Initialize(). Updated the  comment to clarify.